es/10_1_Understanding_the_Foundation_of_P2SH.md

```diff
@@ -0,0 +1,99 @@
+# 10.1: Comprender la Base de P2SH
```

Comment:
I think that the correct translation of `Understanding` is `comprendiendo` in this context.